packages/eds-core-react/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -195,6 +195,28 @@ const findPrevIndex: IndexFinderType = ({
   return prevIndex
 }
 
+type OptionLabelProps<T> = T extends string
```

Comment:
Just thinking of one case, we do support options also being raw numbers without needing optionsLabel, is that possible to incorporate here? (to avoid creating a breaking change)